lib/inertia_rails/ignore_first_load_prop.rb

```diff
@@ -1,5 +1,5 @@
 module InertiaRails
-  class Lazy
+  class IgnoreFirstLoadProp
```

Comment:
I don't think we should rename `Lazy`, let's create a base class `IgnoreFirstLoadProp` instead to mimic what Laravel's plugin does with the interface?

Reply:
I like the idea too, IgnoreFirstLoad too big, and i think optional is the new nomeclature